helix-core/src/doc_formatter.rs

```diff
@@ -127,9 +182,6 @@ pub struct DocumentFormatter<'t> {
     line_pos: usize,
     exhausted: bool,
 
-    /// Line breaks to be reserved for virtual text
-    /// at the next line break
-    virtual_lines: usize,
```

Comment:
Just noticed but the next line is misnamed: `inline_anntoation_graphemes`

Reply:
Good catch, since this is an existing typo I fixed this in a separate commit: [2f277dc](https://github.com/helix-editor/helix/pull/6417/commits/2f277dc7db11b40b4d8f3c281aa0be4718aaa506)